app/Helpers/UserStorage.php

```diff
@@ -61,10 +61,10 @@ public static function getUserById(string $id): mixed
         $users = static::getUsers();
 
         $users = array_filter($users, function ($user) use ($id) {
-            return $user->id === $id;
+            return (int)$user->id === (int) $id;
```

Comment:
This might not work properly,  you might need to do `((int)$user->id) === ((int) $id)`